terragrunt/modules/release-distribution/fastly-static.tf

```diff
@@ -115,6 +115,26 @@ resource "fastly_service_vcl" "static" {
     VCL
   }
 
+  snippet {
+    # This was an abandoned copy of Rust 1.17.0-era docs; redirect to current docs
+    name    = "redirect /doc/master to /stable"
+    type    = "error"
```

Comment:
The snippet will not be executed, since we never make it to the `error` state in the [VCL lifecycle](https://www.fastly.com/documentation/guides/vcl/using/). That's why the existing snippet that redirects `rustup.sh` first has a snippet that detects requests to that endpoint and then fails them with a custom error that we're handling manually.

```vcl
snippet {
  name    = "detect rustup.sh requests"
  type    = "recv"
  content = <<-VCL
    if (req.url ~ "^\/rustup\.sh$") {
      error 618 "redirect";
    }
  VCL
}
```

We need something like this for the docs as well. I suggest that we use a new error code (e.g. `619`), and then replicate how `rustup.sh` is handled.